src/linear_regression_model/TaxRateSlider.jsx

```diff
@@ -74,6 +74,10 @@ export const TaxRateSlider = ({taxRateIdx,
                                     value={(taxRateIdx + 1)}
                                     onChange={(val) => {
                                         handleTaxRateIdx(val -1);}}/>
+                                <div className='invalid-feedback'>
```

Comment:
Is it possible to state the number and what the range is? Something like `-1 is outside the dataset range (1-100).